app/assets/stylesheets/application.css.scss

```diff
@@ -91,9 +96,50 @@ body[data-action="devise/sessions-new"], body[data-action="devise/sessions-creat
   }
 
   body {
+    background-color: $bg-color;
     margin: 0 0 20px;
   }
 
+  form {
+    background: $white;
+    border-radius: 5px;
+    border-collapse: inherit;    
+    border: 1px solid $border;
+  }
+
+  #login-box form{
```

Comment:
Avoid using id selectors<br>Opening curly brace `{` should be preceded by one space